src/vmm/src/vstate.rs

```diff
@@ -20,7 +20,11 @@ use arch::aarch64::gic::GICDevice;
 #[cfg(target_arch = "x86_64")]
 use cpuid::{c3, filter_cpuid, t2, VmSpec};
 #[cfg(target_arch = "x86_64")]
-use kvm_bindings::{kvm_pit_config, CpuId, KVM_MAX_CPUID_ENTRIES, KVM_PIT_SPEAKER_DUMMY};
+use kvm_bindings::{
```

Comment:
You could unify this `use` with the next one (they both `use` things from `kvm_bindings`), then isolate what's `x86_64` specific to avoid new build warnings on `aarch64`.